src/file-1.xoo

```diff
@@ -0,0 +1,17 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonarcloud-github-static-resources/v1/common/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:44259/organizations/org0/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:44259/project/issues?id=sample&pullRequest=6994&issues=AWleRni2Cti0lzGA-U3Z&open=AWleRni2Cti0lzGA-U3Z)

[ISSUE_KEY]: AWleRni2Cti0lzGA-U3Z